src/components/introduction-v2.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import SubmissionsObjects from './submissions-objects';
+
+class Introduction extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2 className='ds-h2'>Quality Payment Program Submissions API Will Close for Official Reporting March 31, 2018</h2>
+        <p className='ds-text--lead'>Performance data for the 2017 Quality Payment Program (QPP) must be submitted by March 31, 2018. After this date, 2017 Qualified Registries and Qualified Clinical Data Registries who have an API token will have limited access to specific endpoints of the Submissions API.</p>
+        <h2 className='ds-h2'>API Access After the 2017 QPP Submission Window Closes</h2>
+        <p className='ds-text'>After March 31, 2018, 2017 Qualified Registries and Qualified Clinical Data Registries who have an API token will not be able to access <b>write</b> and <b>update</b> endpoints in the production environment. In other words, the following endpoints, which were available during the submission window, will no longer be accessible:</p>
+
+        MeasurementSets
+        <ul>
```

Comment:
Hey Sheena, the url formats don't match what Ivana changed to in the document, which is how the /score preview and /score ones are formatted in the paragraph below. It looks like Ivana would prefer these spelled out.

Reply:
other than that looks good to me!